homeassistant/components/withings/config_flow.py

```diff
@@ -192,25 +197,45 @@ class WithingsAuthCallbackView(HomeAssistantView):
 
     requires_auth = False
 
-    def __init__(self, profile: str):
+    def __init__(self, flow_id: str, profile: str):
         """Constructor."""
-        self.profile = profile
-        self.url = auth_callback_path(profile)
-        self.name = auth_callback_name(profile)
+        self._profile = profile
+        self._flow_id = flow_id
+        self._url = auth_callback_path(profile)
+        self._name = auth_callback_name(profile)
+
+    @property
+    def profile(self):
+        """Return profile."""
+        return self._profile
+
+    @property
+    def flow_id(self):
+        """Return flow id."""
+        return self._flow_id
+
+    @property
+    def url(self):
+        """Return url."""
+        return self._url
+
+    @property
+    def name(self):
+        """Return name."""
+        return self._name
 
     @callback
     def get(self, request):
         """Receive authorization code."""
         hass = request.app['hass']
         if 'code' in request.query:
             hass.async_create_task(
-                hass.config_entries.flow.async_init(
-                    const.DOMAIN,
-                    context={'source': const.CODE},
-                    data={
+                hass.config_entries.flow.async_configure(
```

Comment:
Why did we change to `async_configure`?

Reply:
So it would reuse the existing flow_id rather than starting a new flow. I thought that may work better in the future. Although there isn't any docs on the subject so it's hard to say.

Reply:
How does it work now?

Reply:
It works the same as before, the integration gets setup normally. Since the click link step has no way for the user to advance other than clicking the link, I was hoping that by doing it this way, the UI would advance the click link step. It did not.